analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpMethodParameterLookup.cs

```diff
@@ -31,12 +31,12 @@ internal class CSharpMethodParameterLookup : AbstractMethodParameterLookup<Argum
         }
 
         public CSharpMethodParameterLookup(ArgumentListSyntax argumentList, SemanticModel semanticModel)
```

Comment:
Maybe these constructors could be single line.

Reply:
The 2nd one has some logic that should be on a separate line. So I'll keep the 1st one the same for consitency.